src/components/ShareTrajectoryModal/style.css

```diff
@@ -0,0 +1,106 @@
+.upload-modal :global(.ant-modal-footer) {
+    padding: 12px 16px 20px;
+    background-color: var(--modal-content-bg);
+    max-height: 56px;
+}
+
+.upload-modal :global(.ant-modal-body) {
+    padding: 24px 24px 0px 24px;
+    font-size: 16px;
+    color: var(--dark);
+}
+
+.upload-modal :global(.ant-modal-header) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-modal :global(.ant-modal-title) {
+    font-size: 14px;
+}
+
+.upload-modal :global(.ant-modal-close-x) {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
this seems like a lot of additional styling. I haven't looked at the designs in a while but are there really that many differences between our already created modals and this one? My thinking is that all our modals should be styled consistently so things like the checkbox and header should be in the customModals and not just for this modal

Reply:
ok, i've moved a lot of these rules to CustomModal and light-theme